CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # v0.9.2 (Tue Nov 26 2019)
 
-#### 🐛  Bug Fix
+#### 🐛 Bug Fix
 
 - `@jimp/cli`, `@jimp/core`, `@jimp/custom`, `jimp`, `@jimp/plugin-blit`, `@jimp/plugin-blur`, `@jimp/plugin-circle`, `@jimp/plugin-color`, `@jimp/plugin-contain`, `@jimp/plugin-cover`, `@jimp/plugin-crop`, `@jimp/plugin-displace`, `@jimp/plugin-dither`, `@jimp/plugin-fisheye`, `@jimp/plugin-flip`, `@jimp/plugin-gaussian`, `@jimp/plugin-invert`, `@jimp/plugin-mask`, `@jimp/plugin-normalize`, `@jimp/plugin-print`, `@jimp/plugin-resize`, `@jimp/plugin-rotate`, `@jimp/plugin-scale`, `@jimp/plugin-shadow`, `@jimp/plugin-threshold`, `@jimp/plugins`, `@jimp/test-utils`, `@jimp/bmp`, `@jimp/gif`, `@jimp/jpeg`, `@jimp/png`, `@jimp/tiff`, `@jimp/types`, `@jimp/utils`
-  - Fix regeneratorRuntime errors [#815](https://github.com/oliver-moran/jimp/pull/815) ([@SaWey](https://github.com/SaWey) [@hipstersmoothie](https://github.com/hipstersmoothie))
+  - Fix regeneratorRuntime errors [#815](https://github.com/oliver-moran/jimp/pull/815) ([@crutchcorn](https://github.com/crutchcorn) [@hipstersmoothie](https://github.com/hipstersmoothie))
```

Comment:
Since this PR seems to be being worked on, there's other instances in the CHANGELOG that are not properly attributed in the 0.8.0 releases if you're able to fix those too 😅

Reply:
👍

Reply:
https://github.com/oliver-moran/jimp/releases/tag/v0.8.0 seems fine

Reply:
It's `0.8.5` speicifically for the `internal`